.gitattributes

```diff
@@ -9,3 +9,5 @@ src/ncep/xx120 filter=lfs diff=lfs merge=lfs -text
 *.bufr_d filter=lfs diff=lfs merge=lfs -text
 *.grib filter=lfs diff=lfs merge=lfs -text
 *.grib2 filter=lfs diff=lfs merge=lfs -text
+test/testinput/bufr_tables/** filter=lfs diff=lfs merge=lfs -text
```

Comment:
what are these bufr tables and where are they coming from?
Are these copies from nceplibs-bufr? Somewhere else?
I don't think bufr tables belong in the ioda-converters repo.

Reply:
How else will you test this new feature?

Reply:
These tables are necessary in order to open the Std or WMO BUFR files. The data from these tables is not included in those files and must be side loaded.

Reply:
can we add these tables in the `NCEPLIBS-bufr` repo.  That repo contains tables [here](https://github.com/NOAA-EMC/NCEPLIBS-bufr/tree/develop/tables). What are those? And are they different than these? I can see Jeff Ator not wanting to add to his maintenance burden.  We can propose. On the other hand, I don't think the burden of maintaining WMO bufr tables should fall on this repo either.

Reply:
Or maybe just the table(s) necessary to read the one test file we have for the unit test? I think it should be self contained here for the unit testing capability, after all we have numerous .nc, .bufr, etc. files in this repo that are fairly specific.

Reply:
I looked in the nceplibs-bufr repo and most of these tables are identical there.
I suggest we consolidate.  
nceplibs-bufr can read any "std" bufr file by providing an external table.

Reply:
So it turns out Jeff added those tables to their repo very recently, and they are being installed in in a tables directory when you do the `make install`. I guess the question is how we could figure out where this directory was (wish NCEPLIB-bufr would just read the tables automatically). I'll ask Jeff.

The BUFR message has in its message header information about what version of bufr table to use (both WMO and NCEP maintain slightly different versions of these tables). NCEPLIB-bufr looks for tables that have the correct name based on this info (can't just consolidate tables without code changes to NCEPLIB-bufr). Our code simply points it to the directory where the tables are stored.

Reply:
In CMake is there any way to get the install path for a package (bufr in my case) that has been previously found with find_package? Tried a bunch of things... but no dice. My goal is to get the **<bufr install path>/tables** directory and symlink it into the testinput directory at build time...

Reply:
I will tag @mmiesch as he may know better.  Also, a quick question. I have been using your own repo for bufrlib, but will your own mods go back into the master branch of NCEP's bufr?

Reply:
The WMO related updates are actually there now.

Reply:
@rmclaren @aerorahul nceplibs-bufr should make a small change to their [PackageConfig.cmake.in file](https://github.com/NOAA-EMC/NCEPLIBS-bufr/blob/develop/cmake/PackageConfig.cmake.in) to also provide the path to the tables installation. [See here](https://cmake.org/cmake/help/latest/manual/cmake-packages.7.html#id14) for an example. By defining ```set(bufr_TABLES_DIR ${PREFIX}/tables)``` in the package config file, a downstream consumer can use this variable when making the symlink.

Reply:
Thanks for the tip. Will open a PR shortly.

Reply:
Having symbolic links all over the place is messy and prevents us from having independent, relocatable packages. Of course, environment variables are also equally problematic.

Really, though, jedi-wide we need a good policy on how to manage resource data. Similar issues occur in ufo, rttov, crtm, etc. whenever coefficient files are used. These coefficients are *not* test data and are required at runtime. @ytremolet & @mmiesch this is definitely a discussion worth having.